docs/guides/development/data-synchronization.md

```diff
@@ -0,0 +1,169 @@
+
+> 🚧 Product data in the request
+>
+> Unlike customer data, product data provided in the request payload alone do not create a new product in Voucherify. If you want to store products in Voucherify, they need to be created manually in the dashboard or imported with a CSV file.
+> 
+> Also, even though Voucherify uses the payload data instead of the data stored in the system, the product data is not upserted through requests. Products can be updated through product edit function or by importing a CSV file.
+>
+> To enable product updates through the payload, use the [override attribute](#overriding-product-data).
+
+The products in Voucherify can be grouped into static or dynamic collections based on their parameters. Collections can be used in validation rules to model product-specific limits and loyalty-earning rules.
+
+### Product synchronization options
+
+#### Importing products
+
+You can [import your product database](https://support.voucherify.io/article/515-products#import-products-skus "Import Products and SKUs by CSV") to Voucherify with a CSV file. This method can be also used to update the names of existing products.
+
+#### Creating and updating products via API
+
+You can create and update your products via the following API endpoints:
+- [Create product](ref:create-product)
+- [Update product](ref:update-product)
+
+#### Overriding product data
+
+In a request payload, you can add an attribute `override` with value `true` to the product object.
+
+The `override` attribute is used to enable storing the product data in Voucherify. If the product does not exist, it will be created with a `source_id`. If the product does exist, the provided values for the name, price, and metadata will replace those already stored in Voucherify.
+
+## Order synchronization
+
+Orders are synchronized with Voucherify automatically during redemption. They can be viewed through the [Orders view](https://support.voucherify.io/article/263-how-can-i-track-customer-orders "How can I track customer orders?") in the dashboard.
+
+If you want to synchronize the orders that have not been included in any redemption, you need to use the [import order](#import-orders) method.
+
+### Order synchronization options
+
+#### Create order
+
+The [create order](ref:create-order) endpoint creates an order object and triggers an order creation event.
+
+The endpoint can be also used to upsert order data if the order `id` or `source_id` are provided.
+
+#### Update order
+
+The [update order](ref:update-order) endpoint updates the specified order with the values of the parameters in the payload.
+
+#### Import orders
+
+The [import orders](ref:import-orders) endpoint is used to import only historical orders into Voucherify. For on-going synchronization, the [Create order](ref:create-order) and  [update order](ref:update-order) endpoints should be used.
```

Comment:
It's worth to mention, that importing orders in bulk won't trigger distributions and earning rules.

Reply:
I've added that sentence right at the end of this paragraph